misk/src/main/kotlin/misk/web/actions/NotFoundAction.kt

```diff
@@ -38,4 +50,35 @@ class NotFoundAction @Inject constructor() : WebAction {
       )
     }
   }
-}
+
+  /** Generates a message for a client that hints what may have been wrong with their request. */
+  private fun notFoundMessage(path: String, httpCall: HttpCall) = buildString {
+    val boundActions = servletProvider.get().boundActions
```

Comment:
Is it easy to get routes that match your own level of authentication

Reply:
It's not :(